src/Command/RunCommand.php

```diff
@@ -178,6 +178,7 @@ private static function getStepPollDelayFunction(): Closure
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $jobId = (string) getenv('JOB_ID');
+        $storageApiToken = (string) getenv('STORAGE_API_TOKEN');
```

Comment:
Nechci byt hnidopich, ale jelikoz je to Symfony appka, bylo by lepsi to ziskat pres kontejner (ano, vidim i to JOB_ID vys :slightly_smiling_face:)

Reply:
Máš nevšední dar každou sebemenší úpravu zkomplikovat do nějaké nory